olive/model/utils/hf_utils.py

```diff
@@ -188,3 +189,11 @@ def get_model_max_length(model_name: str, fail_on_not_found=False) -> int:
             else:
                 logger.warning(not_found_msg)
                 return None
+
+
+def get_loading_args(hf_config: HfConfig):
+    return hf_config.from_pretrained_args.get_loading_args() if hf_config.from_pretrained_args else {}
+
+
+def get_model_type(hf_config: HfConfig) -> str:
```

Comment:
we need decouple this function with HfConfig by:
1. use model_name,
2. use from_pretrained_args fetched from HfConfig in conversion

Reply:
something like:
```python
def get_model_type(model_name, loading_args) -> str:
```

Reply:
Doesn't that mean every site that calls this function will have to get the model_name and explicitly call to get loading args merely to pass it down to this function. Seems like a lot of unnecessary code duplication.